repose-aggregator/components/filters/body-extractor-to-header-filter/src/test/scala/org/openrepose/filters/bodyextractortoheader/BodyExtractorToHeaderSchemaTest.scala

```diff
@@ -20,17 +20,20 @@
 package org.openrepose.filters.bodyextractortoheader
 
 import org.junit.runner.RunWith
+import org.mockito.Mockito._
 import org.openrepose.commons.test.ConfigValidator
+import org.openrepose.core.services.config.ConfigurationService
 import org.scalatest.junit.JUnitRunner
 import org.scalatest.{FunSpec, Matchers}
 
 @RunWith(classOf[JUnitRunner])
 class BodyExtractorToHeaderSchemaTest extends FunSpec with Matchers {
-  val validator = ConfigValidator("/META-INF/schema/config/body-extractor-to-header.xsd")
+  val filter: BodyExtractorToHeaderFilter = new BodyExtractorToHeaderFilter(mock(classOf[ConfigurationService]))
```

Comment:
There may even be a way to get rid of the redundancy by splitting out some of the configuration stuff from the `AbstractConfiguredFilter` into a trait. Scala objects can extend traits, so we get can define things statically, but then the `AbstractConfiguredFilter` would need to take that information on construction. Something like:
```
class MyFilter extends AbstractConfiguredFilter(MyFilter) {
  ...
}

object MyFilter extends ConfiguredFilterPaths {
  final val SchemaLocation = "/path/to/schema.xsd"
  final val DefaultConfig = "/path/to/default/config.cfg.xml"
}
```